File: Counting Elements/PermCheck/PermCheck.js
function solution(A) {
    A.sort();
    if(A[0] !== 1) {
        return 0;
    }
    for(let i=A.length-1; i>=1; i--) {
        let diff = A[i] - A[i-1];
        if(diff !== 1) {
            return 0;
        }
    }
    return 1;
}

File: 04 Counting Elements/PermCheck/PermCheck.py
def solution(A):
    A.sort()
    
    x = 0;
    if A[0] != 1: return 0
    
    for i in range(0,len(A)-1):
        if A[i] !=  A[i+1]-1:
            return 0
            
    return 1